pkg/cmd/convert.go

```diff
@@ -62,15 +69,39 @@ func GetValidatedObject(v *viper.Viper, cmd *cobra.Command, out, outerr io.Write
 	}
 
 	var ocObjects []*object.OpenCompose
+	var data []byte
```

Comment:
@tnozicka we need to use `data` outside the if else statement, for passing it to decoder, etc. How do you suggest I do this without declaring this before if else?